tsl/test/src/test_compression.c

```diff
@@ -545,18 +550,22 @@ test_delta3(bool have_nulls, bool have_random)
 	/* Forward decompression. */
 	DecompressionIterator *iter =
 		delta_delta_decompression_iterator_from_datum_forward(PointerGetDatum(compressed), INT8OID);
-	for (int i = 0; i < TEST_ELEMENTS; i++)
+	ArrowArray *bulk_result = delta_delta_decompress_all(PointerGetDatum(compressed), INT8OID);
+	for (int i = 0; i < 1015; i++)
```

Comment:
Anything special about 1015? Regardless, might be good to add a comment why this number was picked.

Reply:
It should be TEST_ELEMENTS, changed. Not sure about the specific number, we had this in the old tests.